2-mandatory/4-stocks.js

```diff
@@ -35,6 +35,21 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
 */
 function getAveragePrices(closingPricesForAllStocks) {
     // TODO
+    let newAverage =[]
+    let sum = 0;
+    for (let i = 0; i < CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS.length; i++) {
```

Comment:
In the function, instead of using `CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS`, we need to use the parameter `closingPricesForAllStocks`, so that we can call the function for different arrays of prices.